src/cljs/rems/catalogue.cljs

```diff
@@ -33,24 +33,27 @@
  (fn [db _]
    (::catalogue db)))
 
-(defn- catalogue-item-title [item language config]
-  (let [title (get-catalogue-item-title item language)]
-    (if (urn-catalogue-item? item)
-      [:a.catalogue-item-link {:href (urn-catalogue-item-link item config) :target :_blank} title " " [external-link]]
-      [:span title])))
+(defn- catalogue-item-title [item language]
+  [:span (get-catalogue-item-title item language)])
+
+(defn- catalogue-item-more-info [item config]
+  (when (urn-catalogue-item? item)
+    [:a.btn.btn-secondary {:href (urn-catalogue-item-link item config) :target :_blank}
+     (text :t.catalogue/more-info) " " [external-link]]))
 
 (defn- catalogue-columns [lang config]
-  {:name {:header #(text :t.catalogue/header)
-          :value (fn [item] [catalogue-item-title item lang config])
-          :sort-value #(get-catalogue-item-title % lang)}
-   :cart {:value (fn [i] [cart/add-to-cart-button i])
-          :sortable? false
-          :class "commands"}})
+  {:name     {:header     #(text :t.catalogue/header)
+              :value      (fn [item] [catalogue-item-title item lang])
+              :sort-value #(get-catalogue-item-title % lang)}
+   :commands {:values    (fn [item] [[catalogue-item-more-info item config]
+                                     [cart/add-to-cart-button item]])
+              :sortable? false
+              :class     "commands"}})
```

Comment:
You can probably remove `:class` as the key of the column goes into the class automatically?